src/coord/rmr/chan.c

```diff
@@ -73,18 +51,11 @@ size_t MRChannel_Size(MRChannel *chan) {
   return ret;
 }
 
-PushErrorMask MRChannel_Push(MRChannel *chan, void *ptr) {
-
-  pthread_mutex_lock(&chan->lock);
-  int rc = 0;
-  if (!chan->open) {
-    rc = CHANNEL_CLOSED;
-    goto end;
-  }
```

Comment:
Of what? It is now OK to add items to the channel after it is closed, as long as you still have a reference to it and it will be freed with the channel.
It's also not possible with the current codebase

Reply:
So we decided to rename `MRChannel_Close` to `MRChannel_Unblock`, as this is the only effect of the function. It is also impossible to push an item to a "closed" channel today, as we call this function only when we know no new replies will arrive